borsh/src/schema/container_ext.rs

```diff
@@ -1,8 +1,8 @@
 use super::{BorshSchemaContainer, Declaration, Definition, Fields};
 
-use max_size::is_zero_size;
-pub use max_size::SchemaMaxSerializedSizeError;
-pub use validate::SchemaContainerValidateError;
+pub use max_size::Error as SchemaMaxSerializedSizeError;
+use max_size::{is_zero_size, ZeroSizeError};
+pub use validate::Error as SchemaContainerValidateError;
```

Comment:
it appears to be too verbose in its own module, where the context of which `Error` is meant is more narrow